app/script/conversation/ConversationRepository.js

```diff
@@ -2064,7 +2068,11 @@ z.conversation.ConversationRepository = class ConversationRepository {
    */
   _wrap_in_ephemeral_message(generic_message, millis) {
     const ephemeral = new z.proto.Ephemeral();
-    ephemeral.set('expire_after_millis', millis);
+    const fixedTimer = Math.min(
```

Comment:
@gregor ok, quick discussion about this. Do you think the conversation repo is the right place for this logic?

My argument for adding the logic here, is to have the dumbest entity classes (the anemic domain model pattern which is not loved by everyone, including Martin Fowler :D https://martinfowler.com/bliki/AnemicDomainModel.html but helps separating raw data from logic in my opinion.)

If you have any opinion on this, I am all ears